app/models/user.rb

```diff
@@ -12,6 +12,13 @@ class User < ActiveRecord::Base
 
   has_secure_password
   enum role: %w(default admin)
+  
+  has_attached_file :avatar, styles: {thumb: '100x100>',
+                                      square: '200x200#',
+                                      medium: '300x300>'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.